app/services/message_formatter.rb

```diff
@@ -0,0 +1,21 @@
+class MessageFormatter
+  def initialize(message)
+    @message = message
+  end
+
+  def escape_slack_characters
+    escaped_body = message_body
+    escaped_body = escaped_body.gsub('&', '&amp;')
+    escaped_body = escaped_body.gsub('<', '&lt;')
+    escaped_body = escaped_body.gsub('>', '&gt;')
+    escaped_body
+  end
+
+  private
+
+  attr_reader :message
+
+  def message_body
+    message.body || ""
```

Comment:
Indeed there is!  It's a holdover from what I had done earlier and not accounting for that, let's remove that since it's not needed.